src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveAppKeys.scala

```diff
@@ -260,10 +260,10 @@ trait SbtReactiveAppKeys {
   val minikube = InputKey[Unit]("minikube")
 
   /**
-   * If non-empty (default: "application.conf"), all rp-tooling.conf resources in dependencies will be prepended to the
-   * unmanaged application.conf file, or one will be created if none exists. To disable this behavior, specify "".
+   * If non-empty (default: "application.conf"), all resources with the given name in dependencies will be concatenated into an
+   * managed rp-application.conf file. To disable this behavior, specify "".
    */
-  val prependRpConf = SettingKey[String]("rp-prepend-rp-application-conf")
+  val prependRpConf = SettingKey[String]("rp-concat-to-rp-application-conf")
```

Comment:
Probably best to deprecate this key, and replace it with a "rp-concat-to-rp-application-conf" key.